.github/workflows/test.yml

```diff
@@ -75,6 +78,13 @@ jobs:
           count=$(git status sample --porcelain | wc -l)
           echo "count=$count" >> $GITHUB_OUTPUT
 
+      - name: Copy the ArkanaKeys back to Sample app
+        run: |
+          rm -rf ./sample/ios/ArkanaKeys
+          cp -r ./ArkanaKeys ./sample/ios
+          rm -rf ./ArkanaKeys
+          rm -rf ./sample/ArkanaKeys
```

Comment:
hmm, is this path `./sample/ArkanaKeys` correct? Do we have `ArkanaKeys` in the root project folder?